src/server/glib_main_loop_sources.cpp

```diff
@@ -431,9 +431,9 @@ void md::FdSources::add(
 
     g_source_set_callback(
         gsource,
-        reinterpret_cast<GSourceFunc>(&FdContext::static_call),
+        (GSourceFunc)(void*)(&FdContext::static_call),
         fd_context,
-        reinterpret_cast<GDestroyNotify>(&FdContext::static_destroy));
+        (GDestroyNotify)(void*)(&FdContext::static_destroy));
```

Comment:
Ok, I think I see why this complains.

Shouldn't this be reinterpret_cast<GDestroyNotify>(reinterpret_cast<void*>(&FdContext::static_destroy))?

Reply:
OK, verbosity is in. ;)